server/schedule/operator_controller.go

```diff
@@ -847,6 +851,7 @@ func (oc *OperatorController) exceedStoreLimitLocked(ops ...*operator.Operator)
 				return false
 			}
 			if !limiter.Available(stepCost, v) {
+				operator.OperatorExceedStoreLimitCounter.WithLabelValues(desc).Inc()
```

Comment:
only for urgent？

Reply:
urgent priority not limited by store limit.